src/audio/CMakeLists.txt

```diff
@@ -121,6 +124,7 @@ set(asrc_sources asrc/asrc.c asrc/asrc_farrow.c asrc/asrc_farrow_generic.c)
 set(eq-fir_sources eq_fir/eq_fir.c eq_fir/fir.c)
 set(eq-iir_sources eq_iir/eq_iir.c eq_iir/iir.c eq_iir/iir_generic.c)
 set(dcblock_sources dcblock/dcblock.c dcblock/dcblock_generic.c)
+set(crossover_sources crossover/crossover.c crossover/crossover_generic.c eq_iir/iir_generic.c)
```

Comment:
are you sure we need to add iir_generic? Can we not just include the header?

Reply:
In current implementation, yes, otherwise "iir_df2t" will be a undefined symbol on runtime. It's because the implementation of iir_df2t is under libsof_eq_iir.so and it is not linked together with libsof_crossover.so

Reply:
Interesting, can we treat the iir as a library? I am just worried this method will cause the compiler to duplicate the iir code

Reply:
@singalsu can we make IIR and FIR a library ?

Reply:
@lgirdwood Yes, making them a library sounds like good idea if the compiling duplicates the code. I realized now I have exactly the same issue in beamformer component that reuses FIR.

Reply:
There's already src/math directory for sine, decibels, etc. I wonder FIR and IIR as generic and optimized versions could be moved there? Also later e.g. FFT, IFFT.

Reply:
@singalsu ok, move in math sounds fine. They would be selected in Kconfig by EQ, crosover etc.